projects/experimental/directives/button-vertical/button-vertical.style.less

```diff
@@ -0,0 +1,32 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButton][tuiButtonVertical] {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    padding: 0.75rem;
+    grid-gap: 0.5rem;
+    gap: 0.5rem;
+    line-height: 1rem;
+    min-width: 5rem;
+    text-indent: unset;
+    font-size: var(--tui-font-text-s-2);
+
+    &:not([appearance]) {
```

Comment:
What is this for?

Reply:
ButtonVertical has original appearance by default (at least proprietary), it doesn't match by any existing appearance.
So this is for this appearance

Reply:
decided to make a separate appearance in a proprietary version
removed